common/app/views/fragments/commercial/contentLogo.scala.html

```diff
@@ -1,11 +1,10 @@
 @(branding: common.commercial.Branding)
 
-<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge--interactive">
-    <div class="brandbadge__inner">
+<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge ad-slot--paid-for-badge--front">
```

Comment:
And [here](https://github.com/guardian/frontend/blob/3be9765e2b52c887909ccb0b4dc11b6101c6ec30/commercial/app/views/contentapi/items.scala.html#L85)